include/net/net_context.h

```diff
@@ -87,8 +89,10 @@ typedef void (*net_context_recv_cb_t)(struct net_context *context,
  * @typedef net_context_send_cb_t
  * @brief Network data send callback.
  *
- * @details The send callback is called after a network data is
- * sent.
+ * @details The send callback is called after a network data is sent.
```

Comment:
same here, this should read either "after network data is sent" or maybe "after a network data packet is sent"